testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/postgres/PostgresPolicy.scala

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.bluebrain.nexus.testkit.postgres
+
+import cats.effect.{IO, Timer}
+import cats.implicits._
+import doobie.util.transactor.Transactor
+
+import scala.concurrent.duration.FiniteDuration
+
+import scala.concurrent.duration._
+
+object PostgresPolicy {
+
+  def waitForPostgresReady(xa: Transactor[IO], maxDelay: FiniteDuration = 30.seconds)
```

Comment:
Probably we can get rid of this and use instead the provided izumi check (I think it was recently introduced) directly where needed:
```scala
import izumi.distage.docker.healthcheck.ContainerHealthCheck
ContainerHealthCheck.postgreSqlProtocolCheck(port, username, password)
```